core/banking-protocol/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM openjdk:8-jre-alpine
+
+ARG JAR_FILE
+ENV JAR_FILE ${JAR_FILE}
+ENV APP_HOME /usr/app
+
+WORKDIR $APP_HOME
+
+COPY target/${JAR_FILE} ${JAR_FILE}
+
+EXPOSE 8080
+
+ENTRYPOINT ["sh", "-c", "java -jar ${JAR_FILE}"]
```

Comment:
better use 
`ENTRYPOINT ["sh", "-c", "java", "-jar", "${JAR_FILE}"]`
to avoid issues with spaces

Reply:
ok

Reply:
suggested change doesn't work.
It runs java command without arguments

Reply:
```
FROM openjdk:8-jre-alpine

ARG JAR_FILE
ENV JAR_FILE ${JAR_FILE}
ENV APP_HOME /usr/app

WORKDIR $APP_HOME

COPY target/${JAR_FILE} ${JAR_FILE}
RUN echo "java -jar '${JAR_FILE}'" > run.sh
RUN chmod +x run.sh

EXPOSE 8080

ENTRYPOINT ["sh", "-c", "./run.sh"]
```
Is clearer about what you are running or if you want to stick to yours scenario:
```
FROM openjdk:8-jre-alpine

ARG JAR_FILE
ENV JAR_FILE ${JAR_FILE}
ENV APP_HOME /usr/app

WORKDIR $APP_HOME

COPY target/${JAR_FILE} ${JAR_FILE}

EXPOSE 8080

ENTRYPOINT ["sh", "-c", "java -jar \"${JAR_FILE}\""]
```

Reply:
done